pwa/src/templates/templateParts/layerAccordion/LayerAccordionTemplate.module.css

```diff
@@ -9,3 +9,67 @@
 .items > *:not(:last-child) {
   margin-inline-end: var(--web-app-size-sm);
 }
+
+/* Colors of the 5 layers */
+.interactieCategory,
+.interfaceCategory {
+  background-color: var(--web-app-layer-category-color-intraction) !important;
+  color: var(--web-app-color-black) !important;
+  border: none !important;
+}
+.interactieCategory:hover,
+.interfaceCategory:hover {
+  background-color: var(--web-app-layer-category-color-intraction) !important;
+  color: var(--web-app-layer-color-intraction) !important;
+  cursor: pointer !important;
+}
+
+.procesCategory,
+.processCategory {
+  background-color: var(--web-app-layer-category-color-process) !important;
+  color: var(--web-app-color-black) !important;
+  border: none !important;
+}
+.procesCategory:hover,
+.processCategory:hover {
+  background-color: var(--web-app-layer-category-color-process) !important;
+  color: var(--web-app-layer-color-process) !important;
+  cursor: pointer !important;
+}
+
+.integratieCategory,
+.integrationCategory {
+  background-color: var(--web-app-layer-category-color-integration) !important;
+  color: var(--web-app-color-black) !important;
+  border: none !important;
+}
+.integratieCategory:hover,
+.integrationCategory:hover {
+  background-color: var(--web-app-layer-category-color-integration) !important;
+  color: var(--web-app-layer-color-integration) !important;
+  cursor: pointer !important;
+}
+
+.servicesCategory,
+.serviceCategory {
+  background-color: var(--web-app-layer-category-color-services) !important;
+  color: var(--web-app-color-black) !important;
+  border: none !important;
+}
+.servicesCategory:hover,
+.serviceCategory:hover {
+  background-color: var(--web-app-layer-category-color-services) !important;
+  color: var(--web-app-layer-color-services) !important;
+  cursor: pointer !important;
+}
+
+.dataCategory {
+  background-color: var(--web-app-layer-category-color-data) !important;
+  color: var(--web-app-color-black) !important;
+  border: none !important;
+}
+.dataCategory:hover {
+  background-color: var(--web-app-layer-category-color-data) !important;
+  color: var(--web-app-layer-color-data) !important;
+  cursor: pointer !important;
+}
```

Comment:
I think we should be able to override design tokens here, instead of using the `!important` statement everywhere.

Something like:

```
.processCategory {
  --web-app-current-color-rule: var(--web-app-layer-category-color-process);
  ...
}
```
Also, if we add a generic className to the different buttons, we can remove the duplicated rules (cursor and border) and add them to the generic className (since they're the same everywhere).